VirtoCommerce.Platform.Web/Scripts/app/dynamicProperties/blades/dynamicProperty-detail.tpl.html

```diff
@@ -88,7 +88,7 @@
                         <input class="form-input" num-type="integer" type="number" ng-min="0" min="0" ng-model="blade.currentEntity.displayOrder" placeholder="{{ 'platform.blades.dynamicProperty-detail.placeholders.display-order' | translate}}" />
                     </div>
                 </div>
-                <div class="form-group">
+                <div class="form-group" hidden>
```

Comment:
Need just remove this input if it no longer needed

Reply:
This field was needed because at update of field earlier old description could be lost. Now this behaviour is not observing. This field will be remove.